### PR TITLE
Document Security Infrastructure

### PR DESCRIPTION
**Pull request by ElizabethStirling**
Brainstate dump of the infrastructure and logging work I've been doing recently. I definitely should have been doing this incrementally.

Please review this through the lens of "What question have I had for Elizabeth that this doesn't answer?"

### CODE REVIEW

File: handbook/engineering/security/infrastructure/index.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
+# Security Infrastructure
+
+We maintain multiple flavors of infrastructure with various degrees of management. 
+
+
+
+## GCP Infrastructure Basics
```
Comment (unknwon) [inline handbook/engineering/security/infrastructure/index.md:7]: [We favor sentence cases](https://about.sourcegraph.com/handbook/communication/content_guidelines/actionable_language#headings-and-subheadings) :D

File: handbook/engineering/security/infrastructure/playbooks.md [hunk 2]
```diff
@@ -0,0 +1,372 @@
...
+2. Get access to the logging cluster by running `gcloud container clusters get-credentials --region us-central1-f --project sourcegraph-security-logging logging`
+3. From `security/logging/helm/pubsubbeat` run the command `helmfile template | kubectl --context gke_sourcegraph-security-logging_us-central1-f_logging apply -f -`
+
+### Logging Staging
+
+To implement in [#17281](https://github.com/sourcegraph/sourcegraph/issues/17281).
+
+## GCP Deployment Playbooks
+
+Playbooks and step - by - step instructions on how to deploy our projects. It's assumed that you already have [terraform](https://www.terraform.io/) installed.
```
Comment (unknwon) [inline handbook/engineering/security/infrastructure/playbooks.md:31]:
```suggestion
Playbooks and step-by-step instructions on how to deploy our projects. It's assumed that you already have [terraform](https://www.terraform.io/) installed.
```

**Review by unknwon — APPROVED** [review]
Thanks for writing these docs!

Would also love a review pass by @quinnkeast 😁

**Review by chayim — APPROVED** [review]
Huge improvement. I love this. You even answered my encryption question, prior to my asking it.

**Review by andreeleuterio — APPROVED** [review]

File: handbook/engineering/security/infrastructure/index.md (hunk 1 above)
Comment (ElizabethStirling) [reply handbook/engineering/security/infrastructure/index.md:7]: Good catch, thanks so much!

**Pull request merged**